pkg/util/utils_supported.go

```diff
@@ -60,6 +60,36 @@ func GetRootlessRuntimeDir() (string, error) {
 	return rootlessRuntimeDir, nil
 }
 
+// GetRootlessConfigHomeDir returns the config home directory when running as non root
+func GetRootlessConfigHomeDir() (string, error) {
+	var rootlessConfigHomeDirError error
+
+	rootlessConfigHomeDirOnce.Do(func() {
+		home := os.Getenv("HOME")
```

Comment:
we can move reading "HOME" in the if block when `XDG_CONFIG_HOME` is empty.

So we won't fail early if `HOME` is not set but `XDG_CONFIG_HOME` is.

Reply:
Done